jetty-core/jetty-compression/jetty-compression-brotli/pom.xml

```diff
@@ -23,14 +23,9 @@
       <artifactId>brotli4j</artifactId>
       <version>${brotli4j.version}</version>
     </dependency>
-    <dependency>
-      <groupId>io.netty</groupId>
-      <artifactId>netty-buffer</artifactId>
-      <version>${netty.version}</version>
-    </dependency>
     <dependency>
       <groupId>org.eclipse.jetty.compression</groupId>
-      <artifactId>jetty-compression-api</artifactId>
+      <artifactId>jetty-compression-common</artifactId>
       <version>${project.version}</version>
```

Comment:
Like all other Jetty modules, this should be declared in `<dependencyManagement>` in the main POM, so that here you don't need to specify the version.
Same for all the other sub-modules.